de/html/README.md

```diff
@@ -91,9 +91,9 @@ Der Titel "Olas Blog" wird nun im Browser angezeigt. Hast du es bemerkt? Der Bro
 
 Wie du vielleicht bemerkt hast, hat jedes Element zu Beginn einen öffnenden Tag und einen zugehörigen *schließenden Tag* mit `/` und innerhalb davon sind Elemente *eingebettet*. Ein innerer Tag kann nicht außerhalb des umschließenden Tags geschlossen werden, die Reihenfolge muss immer stimmen.
 
-Es ist wie, wenn man Sachen in Kisten steckt. In die größte Box: `<html></html>` kommt eine kleinere Box: `<body></body>`, und dort wiederum packen wir viele kleine Kisten hinein: `<p></p>`.
+Es ist, wie wenn man Sachen in Kisten steckt. Du hast eine große Kiste, `<html></html>`. In der ist als weitere, etwas kleinere Kiste `<body></body>` drin, und in der wiederum weitere kleine Kistchen: `<p></p>`.
 
```

Comment:
The comma has to be after "ist", as "wie" here is already part of the subclause. This becomes more apparent if you add the implicit "so" back:
> Es ist **so**, wie wenn man Sachen in Kisten steckt.

You can't really substitute "vergleichbar" here without further changes. However, you could use "als ob" instead of "wie wenn", also _after_ the comma (starting the subclause):
> Es ist, **als ob** man Sachen in Kisten steckt.